### PR TITLE
Fix: add TallyHo

### LINKED ISSUE #703: Add Tally Ho as a wallet option
## Overview

In the product sync we have decided to add Tally Ho as a wallet option. In terms of further maintenance & support we limit our focus on the most used wallet options. 

## Requirements

- [ ] Install onboard module

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

Resolves #703

## How this PR fixes it
Adds the TallyHo module.

## Screenshots
Screenshot 2022-10-05 at 09 11 55

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -15,10 +16,10 @@ export const BEAMER_ID = process.env.NEXT_PUBLIC_BEAMER_ID || ''
 
 // Wallets
 export const WC_BRIDGE = process.env.NEXT_PUBLIC_WC_BRIDGE || 'https://safe-walletconnect.gnosis.io/'
-export const FORTMATIC_KEY = process.env.NEXT_PUBLIC_FORTMATIC_KEY || ''
-export const PORTIS_KEY = process.env.NEXT_PUBLIC_PORTIS_KEY || ''
```
Comment (iamacook) [inline src/config/constants.ts:19]: We should check whether we're paying for these at some point as well. We should cancel them if so.

File: src/config/constants.ts [hunk 2]
```diff
@@ -28,10 +29,10 @@ export const SAFE_TOKEN_ADDRESSES: { [chainId: string]: string } = {
 }
 
 // Safe Apps
-export const SAFE_APPS_POLLING_INTERVAL = process.env.NODE_ENV === 'test' ? 4500 : 15000
+export const SAFE_APPS_INFURA_TOKEN = process.env.NEXT_PUBLIC_SAFE_APPS_INFURA_TOKEN || INFURA_TOKEN
+export const SAFE_APPS_POLLING_INTERVAL = 15_000
```
Comment (iamacook) [inline src/config/constants.ts:33]: Why remove the period when polling?

File: src/config/constants.ts [hunk 3]
```diff
@@ -15,10 +16,10 @@ export const BEAMER_ID = process.env.NEXT_PUBLIC_BEAMER_ID || ''
 
 // Wallets
 export const WC_BRIDGE = process.env.NEXT_PUBLIC_WC_BRIDGE || 'https://safe-walletconnect.gnosis.io/'
-export const FORTMATIC_KEY = process.env.NEXT_PUBLIC_FORTMATIC_KEY || ''
-export const PORTIS_KEY = process.env.NEXT_PUBLIC_PORTIS_KEY || ''
-export const TREZOR_APP_URL = 'gnosis-safe.io'
-export const TREZOR_EMAIL = 'safe@gnosis.io'
+export const TREZOR_APP_URL = 'app.safe.global'
```
Comment (iamacook) [inline src/config/constants.ts:19]: Have you tested that this works on staging? Maybe it should be dynamic.

File: src/config/constants.ts (hunk 2 above)
Comment (katspaugh) [reply src/config/constants.ts:33]: It's still there, I just moved the infura var up.

File: src/config/constants.ts (hunk 3 above)
Comment (katspaugh) [reply src/config/constants.ts:19]: It's just for links I believe. As you can see `gnosis-safe.io` has been working fine all along.

File: src/config/constants.ts (hunk 1 above)
Comment (katspaugh) [reply src/config/constants.ts:19]: We're definitely not paying for them.

File: src/config/constants.ts (hunk 2 above)
Comment (iamacook) [reply src/config/constants.ts:33]: `SAFE_APPS_POLLING_INTERVAL` was dependent on the `NODE_ENV`.

Comment (katspaugh) [reply src/config/constants.ts:33]: We don't have that set anywhere, and it seems it didn't affect anything.

Comment (iamacook) [reply src/config/constants.ts:33]: It's used in `useAppIsLoading` for tests but I see that they're still working.

**Comment by iamacook** [comment]
I'll hold off on approving this until we head back from product [here](https://5afe.slack.com/archives/C03DAGWJCR5/p1664952542007989).

File: src/config/constants.ts (hunk 2 above)
Comment (katspaugh) [reply src/config/constants.ts:33]: I meant that NODE_ENV isn't set anywhere.

Comment (iamacook) [reply src/config/constants.ts:33]: Jest [sets it to `"test"` by default](https://jestjs.io/docs/environment-variables#node_env).

Comment (yagopv) [reply src/config/constants.ts:33]: This came from safe-react. Was renamed here (https://github.com/safe-global/safe-react/commit/918d68eb01b918f2994cb26bdc3de912d414560c). I think we can remove the process.env check without problems as there is no specific test for apps not loading.

Comment (yagopv) [reply src/config/constants.ts:33]: We can rename the constant as well as the name perhaps does not make sense

Comment (katspaugh) [reply src/config/constants.ts:33]: Should I rename it to `SAFE_APPS_LOAD_TIMEOUT`?

Comment (yagopv) [reply src/config/constants.ts:33]: For me fine if you want to do it. If not we can do it in one of our PRs

Comment (katspaugh) [reply src/config/constants.ts:33]: I'll leave it as is for now then.

**Review by iamacook — APPROVED** [review]

**Pull request merged**